CHANGES/3351.bugfix

```diff
@@ -0,0 +1 @@
+Fixed bug about malformed tuple introduced on the removal of sqlite-metadata support (PR #3328).
```

Comment:
FWIW, we sometimes mark issues like this as `.misc`, because as the bugs never made it to any users, it's not strictly relevant to the changelog.

But it's fine either way.

Reply:
Oh, ok.